sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,140 @@
+
+General documentation regarding this calibrator can be found [here](../../tag_based_pnp_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../README.md)
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1gFrrchW9mWM1huWMYuJ0nWB2n1BfCJag).
+
+The rosabg includes four different topics including `camera_info`, `image_rect_color/compressed`, `pointcloud_raw`, and `tf_static`.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. For a normal consumer-level car, a space of `5m x 10m` should be sufficient.
+
+### AprilTag (LidarTag)
+
+AprilTag are the only moving elements during the calibration process and must detected by both camera and lidar.
+Depending on the lidar model and the available space, the required AprilTag size may vary, but so far we have had good results with 0.6m and 0.8m tags (the provided sizes correspond to an edge's size. In these cases the payloads are 0.45m and 0.6m).
+
+In addition to the size of the AprilTag, which affects the physical positions where a tag can be detected, the structure on which the AprilTag is mounted is equally important. The shape and size of this structure may affect the lidar detection algorithm. Therefore, it is advisable to use a mount that positions the tag in such a way that it remains invisible to the sensor (refer to the provided example).
+
+## Launching the tool
+
+In this tutorial, we will use the X2 vehicle of Tier IV.
+First, run the sensor calibration manager:
+
+```text
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `x2`, and in `calibrator`, select `tag_based_pnp_calibrator`. Then, press `Continue`.
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/menu1.jpg"  alt="menu1">
+</p>
+
+A menu titled `Launcher configuration` should appear in the UI, and the user may change any parameter he deems convenient.
+For this tutorial, we will modify the default values `calibration_pairs` from `9` to `8` as the bag has 8 AprilTag detections and also modify the `camera_name` from `camera0` to `camera6`. After configuring the parameters, click `Launch`.
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/menu2.jpg"  alt="menu2">
+</p>
+
+The following UI should be displayed. When the `Calibrate` button becomes available, click it.
+If it does not become available, it means that either the required `tf` or services are not available.
+
+In this tutorial, since the `tf` are published by the provided rosbag, run the rag (`ros2 bag play camera_lidar.db3 --clock -r 0.1`) first and launch the tools afterward to trigger the `Calibrate` button.
```

Comment:
Trigger is not the correct word here

Reply:
Thanks! fixed in https://github.com/tier4/CalibrationTools/commit/32dd61431d3beb3d84ecb085cd246042de8efb09

Reply:
In `In this tutorial, since the `tf` are published by the provided rosbag, start`
The use of the comma is incorrect

Reply:
solved in https://github.com/tier4/CalibrationTools/pull/164/commits/14c627267650c7ca3b1a363a3ed4f2f5efa2fd48